lib/update-lockfile.js

```diff
@@ -52,17 +52,18 @@ module.exports = function updateLockfile (dependency, options) {
   if (exec('git status --porcelain').toString() === '') return
 
   // stage the updated lockfile
-  exec('git add npm-shrinkwrap.json 2>/dev/null || true')
-  exec('git add package-lock.json 2>/dev/null || true')
-  exec('git add yarn.lock 2>/dev/null || true')
+  const ignoreOutput = options.ignoreOutput ? options.ignoreOutput : '2>/dev/null || true'
+  exec(`git add npm-shrinkwrap.json ${ignoreOutput}`)
+  exec(`git add package-lock.json ${ignoreOutput}`)
+  exec(`git add yarn.lock ${ignoreOutput}`)
```

Comment:
Here I'm using the ignore output method defined in the ci-service, if its available.